lib/uri-beacon-scanner.js

```diff
@@ -63,29 +63,25 @@ UriBeaconScanner.prototype.parseUriBeacon = function(peripheral) {
 
   var flags = data.readUInt8(0); // flag is the 1st byte
   var txPower = data.readInt8(1); // TX Power is 2nd byte
-  var uriData = data.slice(2); // remainder in the URI
-
+  var firstByte = data[2];
+  var uriData = data.slice(3); // remainder in the URI
   var uriPrefix = '';
```

Comment:
From the spec "The Uri Scheme Prefix byte defines the identifier scheme, an optional prefix and how the remainder of the Uri is encoded."

The "optional" part confuses me.